packages/web-components/tests/e2e-storybook/cypress/plugins/index.js

```diff
@@ -5,8 +5,11 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+require('dotenv').config({ path: `${__dirname}/../../../../.env` });
+
 // eslint-disable-next-line no-unused-vars
 module.exports = (on, config) => {
+  config.env.DDS_CLOUD_MASTHEAD = process.env.DDS_CLOUD_MASTHEAD;
```

Comment:
Might be the 'feature flag' label, but I'm not sure - adding it just in case.

Reply:
@andy-blum I had to remove that code from the `plugins/index.js` file because it was causing the tests on browserstack to bomb. When browserstack takes the tests, it only has access to the cypress folder so it can't read the `.env` file outside of it. What we ended up doing was adding the cloud-masthead test file to the `excluded` field in the browserstack.json config.

Reply:
I did forget to remove the line in the cloud-masthead test file checking for the env var - will remove and run locally to see if that does anything.

Reply:
i'm curious if it was never passing and it was only running the `masthead` e2e tests, so it seems like it was passing...

Reply:
it ended up being that in `cypress.json` the `baseURL` was being set to canary instead of a `localhost`